lstchain/image/muon/muon_analysis.py

```diff
@@ -122,13 +124,36 @@ def fit_muon(x, y, image, geom, tailcuts):
     image_clean: `np.ndarray`
         Image after cleaning
     """
-
+    
+    if tailcuts == None:
+        # We want to quantify the noise of the image. To do so, we will use the 
+        # negative Q cumulative distribution.
+        negative_Q = np.sort(image[image <= 0])
+        
+        hist, bins = np.histogram(negative_Q,range=(-15,0),bins=30)
```

Comment:
why this range from -15?

Reply:
This is in p.e. , and was considered a reasonable value to include the negative fluctuations, based on inspection of moon data. Could of course be made wider.